dev_cga.log

```diff
@@ -1,518 +1,450 @@
-[March 24, 2022, 1:31 pm] SELECT * FROM loggedin WHERE user_digest = ? LIMIT 1 [['user_digest', '09911812cc482990e40979559073e11c']]
-[March 24, 2022, 1:31 pm] SELECT * FROM users WHERE is_instructor = ? [['is_instructor', '1']]
-[March 24, 2022, 1:31 pm] SELECT * FROM course_offerings
```

Comment:
Doesn't the gitignore make it so I dont have to delete it?? 
I'm confused as to what git ignore file does then :(

Reply:
I deleted it

Reply:
gitignore makes it so that whenever you change a certain file, git ignores it. You should have deleted it in the other pull request, then with it being in the gitignore it should never be checked in ever again.

Reply:
oh, I see :(